infra/consul/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM consul:1.7.1
+
+COPY agent-config.json /consul/configs/
+
+ENTRYPOINT ["consul", "agent", "-config-dir=/consul/configs", "-data-dir=/tmp/consul"]
```

Comment:
Is there any way to do this stuff in docker compose file?

Reply:
I did not find way to do that in docker compose file. I had a problem with docker compose to create config for consul agent. Dockerfile was the best solution for me